lib/qa/authorities/linked_data/config/term_config.rb

```diff
@@ -176,7 +176,9 @@ def term_subauthority_replacement_pattern
       def term_url_with_replacements(id, sub_auth = nil, replacements = {})
         return nil unless supports_term?
         sub_auth = sub_auth.to_sym if sub_auth.is_a? String
-        url = Config.replace_pattern(term_url_template, term_qa_replacement_patterns[:term_id], id)
+        # do not encode if term parameter is starts the template (e.g. template="{?term_uri}" OR template="{?term_uri}.rdf")
```

Comment:
@elrayle I'm wondering if this choice would be better left to the user?  Aka passing a encode parameter into the method.  Is there a reason the user would not know if they want to encode?

Reply:
@cam156 That's good feedback.  I will look and see if I can make that happen in the configuration.